app/src/main/java/com/github/dfa/diaspora_android/activity/AboutActivity.java

```diff
@@ -187,6 +231,28 @@ public String getLicense3dPartyHtml(Context context) {
             text = text.replace("NEWENTRY", "<font color='" + accentColor + "'><b>*</b></font> ");
             return text;
         }
+
+        @Override
+        protected void applyColorToViews() {
+            ThemeHelper.getInstance(getAppSettings());
+            ThemeHelper.updateTextViewColor(textLicense3partyBox);
+            ThemeHelper.updateTextViewColor(textLicenseBox);
+        }
+
+        @Override
+        public String getFragmentTag() {
+            return null;
```

Comment:
Done, but unnecessary (We don't handle these Fragments in the FragmentManager anyways).